.github/workflows/cd-pypi.yml

```diff
@@ -16,9 +16,25 @@ on:
         type: string
         required: false
         default: "./"
+      platforms:
+        description: Platforms to build for e.g. "['ubuntu-latest','macos-latest','windows-latest']"
+        required: false
+        type: string
+        default: ''
+      pyversions:
+        description: Python versions to build for e.g. "['38','39','310','311','312']"
+        required: false
+        type: string
+        default: ''
+      build_args:
+        description: Args to pass in to build as `python -m build {build_args}` e.g. '--sdist'
+        type: string
+        required: false
+        default: ''
+
 
 jobs:
-  deploy:
+  deploy_source:
```

Comment:
this is a bit misleading name, I mean, we don't expect `--sdist` to always be given here. Perhaps `deploy_purepython`, and then the other would be `deploy_binary`? Because `deploy_wheels` does produce wheels, but then this one does too...

Reply:
in a similar fashion, I'd prefix the input args with `binwheel_platforms`, `binwheel_pyversions`, `purepython_buildargs`, to make it clear where they end up used